incubator/datadog-apm/values.yaml

```diff
@@ -0,0 +1,80 @@
+# Default values
+
+nameOverride: ""
+fullnameOverride: ""
+targetSystem: linux
+
+datadog:
+  apiKey:  # <DATADOG_API_KEY>
+  apiKeyExistingSecret:   # <DATADOG_API_KEY_SECRET>
+  appKey:   # <DATADOG_APP_KEY>
+  appKeySecretName:   # <DATADOG_APP_KEY_SECRET>
+  logLevel: INFO
+  site: datadoghq.com
+
+clusterAgent:
+  replicas: 2
+  image:
+    repository: datadog/agent
+    pullPolicy: Always
+    # Overrides the image tag which defaults to the chart appVersion, created by datadog
+    tag: 7.23.0
+  command:
+    - trace-agent
+    - -config=/etc/datadog-agent/datadog.yaml
+  enabled: true
+  token:   # <DATADOG_CLUSTER_AGENT_TOKEN>
+  apm:
+    enabled: true
+    nonLocalTraffic: true
+    receiverPort: 8126
+  metricsProvider:
+    enabled: true
+    port: 8126
+    service:
+      type: ClusterIP
+      port: 8126
+    aggregator: avg
+    wpaController: false
+    useDatadogMetrics: false
+  priorityClassName:  # <FW_PRIORITY_CLASS_NAME>
+  serviceAccount:
+    create: true
+    name: datadog-apm
+  healthPort: 5555
+  livenessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 15
+    periodSeconds: 15
+    successThreshold: 1
+    tcpSocket:
+      port: 8126
+    timeoutSeconds: 5
+  readinessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 15
+    periodSeconds: 15
+    successThreshold: 1
+    tcpSocket:
+      port: 8126
+    timeoutSeconds: 5
+  resources:
+    limits:
+      cpu: 50m
+      memory: 150Mi
+    requests:
+      cpu: 50m
+      memory: 150Mi
+  hpa:
+    enabled: true
+    minReplicas: 1
+    maxReplicas: 6
+    averageCpuUtilization: 30
```

Comment:
I think we can default to a higher cpu utilization and also have `enabled` default to false. Since we'll likely be the main users of this chart, I don't think we need an HPA for this.

Reply:
I definitely think we should default to using the HPA. Part of the reason for running this chart is the scaling functionality for us. 

However, I think we need a memory-based autoscaler, since that's our usual failure point

Reply:
To confirm a memory based hpa vs a cpu based hpa or should we have both?

Reply:
Modified HPA cpu to memory based.